src/locate.rs

```diff
@@ -2,8 +2,17 @@ use std::path::PathBuf;
 
 use crate::Result;
 
-pub fn locate_steam_dir() -> Result<PathBuf> {
-    locate_steam_dir_helper()
+pub fn locate_steam_dir() -> Result<Vec<PathBuf>> {
+    #[cfg(target_os = "linux")]
+    return locate_steam_dir_helper();
+    #[cfg(not(target_os = "linux"))]
+    {
+        let result = locate_steam_dir_helper();
+        match result {
+            Ok(path) => Ok(vec![path]),
+            Err(e) => Err(e),
+        }
+    }
```

Comment:
```suggestion
    locate_steam_dir_helper().map(|path| vec![path])
```

Slightly more succinct